harrisonDir/harrisonHomework.txt

```diff
@@ -0,0 +1,113 @@
+PART 1:
+
+a) true, because == treats strings and numbers as being equal
+
+b) false, because === does NOT treat strings and numbers as being equal (i.e. === cares about type)
+
+c) true, for the same reason as a)
+
```

Comment:
Not quite. In this instance we are evaluating variable x to string x. The value of variable x could be anything, but unless it is literally 'x' this problem will always return false.